Updated the Schema Cache limitations (#19374)
pingcap/docs: schema-cache.md

    - `SHOW FULL TABLES`
    - `FLASHBACK`
    - `ALTER TABLE ... SET TIFLASH MODE ...`

> - For tables that use "AUTO_INCREMENT" or "AUTO_RANDOM," if the Schema cache is set too small, these tables may frequently enter and exit the cache (which can be monitored by observing the Schema cache hit rate and size in TiDB Dashboard). This situation may cause allocated ID segments to expire before they are fully used, resulting in ID jumps. In scenarios with high write volumes, it may even lead to ID segment exhaustion. Therefore, it is recommended to increase the Schema cache size to avoid frequent cache evictions, or set AUTO_ID_CACHE=1 to prevent ID jumps. Additionally, it is recommended to properly configure the AUTO_RANDOM sharding bits and reserved bits to avoid an overly small range of assignable IDs.

Reviewer: ```suggestion
- When using tables with `AUTO_INCREMENT` or `AUTO_RANDOM`, a small schema cache size might cause these tables to frequently enter and leave the cache (which can be monitored through the schema cache hit rate and size on the monitoring panel). This can result in the allocated ID range becoming invalid before being fully used, leading to ID jumps. In high-write scenarios, this might even exhaust the ID range. To minimize abnormal ID allocation behavior and improve system stability, consider the following measures:

    - Increase the schema cache size to reduce frequent evictions.
    - Set [`AUTO_ID_CACHE`](/auto-increment.md#auto_id_cache) to `1` to prevent ID jumps.
    - Properly configure the sharding bits and reserved bits for AUTO_RANDOM to avoid having a limited ID range.
```

    - `SHOW FULL TABLES`
    - `FLASHBACK`
    - `ALTER TABLE ... SET TIFLASH MODE ...`

> - When using tables with `AUTO_INCREMENT` or `AUTO_RANDOM`, a small schema cache size might cause these tables to frequently enter and leave the cache (which can be monitored through the schema cache hit rate and size on the monitoring panel). This can result in the allocated ID range becoming invalid before being fully used, leading to ID jumps. In high-write scenarios, this might even exhaust the ID range. To minimize abnormal ID allocation behavior and improve system stability, consider the following measures:

Reviewer: ```suggestion
- When you use tables with `AUTO_INCREMENT` or `AUTO_RANDOM`, a small schema cache size might cause these tables to frequently enter and leave the cache, which can be monitored through the schema cache hit rate and size on the monitoring panel. This can result in the allocated ID range becoming invalid before being fully used, leading to ID jumps. In write-intensive scenarios, this might even exhaust the ID range. To minimize abnormal ID allocation behavior and improve system stability, consider the following measures:
```

> - When you use tables with the [`AUTO_INCREMENT`](/auto-increment.md) or [`AUTO_RANDOM`](/auto-random.md) attribute, a small schema cache size might cause these tables to frequently enter and leave the cache. You can monitor it through the schema cache hit rate and size on the monitoring panel. This can result in the allocated ID range becoming invalid before being fully used, leading to ID jumps. In write-intensive scenarios, this might even exhaust the ID range. To minimize abnormal ID allocation behavior and improve system stability, it is recommended to take the following measures:

Reviewer: ```suggestion
- When you use tables with the [`AUTO_INCREMENT`](/auto-increment.md) or [`AUTO_RANDOM`](/auto-random.md) attribute, a small schema cache size might cause the meta data of these tables to frequently enter and leave the cache. This can result in the allocated ID range becoming invalid before being fully used, leading to ID jumps. In write-intensive scenarios, this might even exhaust the ID range. To minimize abnormal ID allocation behavior and improve system stability, it is recommended to take the following measures:
```

Author: a small schema cache size might cause the metadata of these tables to be frequently cached and evicted in memory.

>     - Increase the schema cache size to reduce frequent evictions.

Reviewer: ```suggestion
    - View the hit rate and size of the schema cache through the monitoring panel to assess whether the cache settings are reasonable. Increase the schema cache size properly to reduce frequent evictions.
```

    - Set [`AUTO_ID_CACHE`](/auto-increment.md#auto_id_cache) to `1` to prevent ID jumps.

>     - Properly configure the sharding bits and reserved bits for AUTO_RANDOM to avoid having a limited ID range.

Reviewer: ```suggestion
    - Properly configure the sharding bits and reserved bits for `AUTO_RANDOM` to avoid having a limited ID range.
```

> - When you use tables with the [`AUTO_INCREMENT`](/auto-increment.md) or [`AUTO_RANDOM`](/auto-random.md) attribute, a small schema cache size might cause these tables to frequently enter and leave the cache, which can be monitored through the schema cache hit rate and size on the monitoring panel. This can result in the allocated ID range becoming invalid before being fully used, leading to ID jumps. In write-intensive scenarios, this might even exhaust the ID range. To minimize abnormal ID allocation behavior and improve system stability, consider the following measures:
> 
>     - Increase the schema cache size to reduce frequent evictions.
>     - Set [`AUTO_ID_CACHE`](/auto-increment.md#auto_id_cache) to `1` to prevent ID jumps.
>     - Properly configure the sharding bits and reserved bits for `AUTO_RANDOM` to avoid having a limited ID range.

Reviewer 2: ```suggestion
- When you use tables with the [`AUTO_INCREMENT`](/auto-increment.md) or [`AUTO_RANDOM`](/auto-random.md) attribute, a small schema cache size might cause these tables to frequently enter and leave the cache. You can monitor it through the schema cache hit rate and size on the monitoring panel. This can result in the allocated ID range becoming invalid before being fully used, leading to ID jumps. In write-intensive scenarios, this might even exhaust the ID range. To minimize abnormal ID allocation behavior and improve system stability, it is recommended to take the following measures:

    - Increase the schema cache size to reduce frequent evictions.
    - Set [`AUTO_ID_CACHE`](/auto-increment.md#auto_id_cache) to `1` to prevent ID jumps.
    - Properly set the shard bits and reserved bits of `AUTO_RANDOM` to avoid a too small ID range.
```
